src/__tests__/field/__snapshots__/custom_input.test.jsx.snap

```diff
@@ -22,6 +22,14 @@ exports[`CustomInput when type == checkbox renders correctly as a CheckBoxInput
 </div>
 `;
 
+exports[`CustomInput when type == hidden renders correctly as a input[type=hidden] 1`] = `
+<input
+  name="custom_input"
```

Comment:
Same here ... order of attributes

Reply:
we can't control how snapshots are rendered